README.md

```diff
@@ -33,15 +33,11 @@ To run in production use `npm start` or `npm run prod` after running the `npm ru
 At this time no actually encoded TC String is used. This service uses an with
 IAB unregistered vendor ID of `4040`.
 
-## Creating a new version
+## Development
```

Comment:
I'd add an extra section on how to propose a new change to the repository as this is the most common use case. E.g. to propose a new feature / fix a bug you create a branch that follows pattern x..